index.shtml

```diff
@@ -113,16 +113,22 @@
 
       <hr/><br/>
 
-      <b>Find us on social media!</b>
-
-      <table>
-          <tr>
-              <td><a href="https://www.facebook.com/groups/genesis.scifi" target="_blank"><img src="images/facebook-logo.svg" alt="Facebook" width="40" height="40"/></a></td>
-              <td><a href="https://twitter.com/Genesis_SciFi" target="_blank"><img src="images/twitter-logo-250x250.png" alt="Twitter" width="40" height="40"/></a></td>
-              <td><a href="https://instagram.com/Genesis_SciFi" target="_blank"><img src="images/instagram-logo-250x250.png" alt="Instagram" width="40" height="40"/></a></td>
-          </tr>
-      </table>
-
+	  <table class="links">
+	    <tr>
+	      <th>Find us on social media!</th>
+	      <th>Donate!</th>
+	    </tr>
+	    <tr>
+	      <td>
+                <a href="https://www.facebook.com/groups/genesis.scifi" target="_blank"><img src="images/facebook-logo.svg" alt="Facebook" width="40" height="40"/></a>
+                <a href="https://twitter.com/Genesis_SciFi" target="_blank"><img src="images/twitter-logo-250x250.png" alt="Twitter" width="40" height="40"/></a>
+                <a href="https://instagram.com/Genesis_SciFi" target="_blank"><img src="images/instagram-logo-250x250.png" alt="Instagram" width="40" height="40"/></a>
```

Comment:
In keeping with my comment below about making the social media and donation button heights the same, I offer the following suggestion:

```suggestion
                <a href="https://www.facebook.com/groups/genesis.scifi" target="_blank"><img src="images/facebook-logo.svg" alt="Facebook" width="50" height="50"/></a>
                <a href="https://twitter.com/Genesis_SciFi" target="_blank"><img src="images/twitter-logo-250x250.png" alt="Twitter" width="50" height="50"/></a>
                <a href="https://instagram.com/Genesis_SciFi" target="_blank"><img src="images/instagram-logo-250x250.png" alt="Instagram" width="50" height="50"/></a>
```